### DISCUSSION TITLE
Build stuck after Cloning completed

### DISCUSSION THREAD

**Discussion by zoultrex** [vercel/community#515] [Help]
I have a build (for prod) that has passed the 45minutes time and all I see on the deploy logs on the portal is:

Cloning github.com....
Cloning completed: 1.041s

Nothing else is printed after all this time
How can I inspect this build to get more details? I have searched here and on the docs but nothing has helped so far

Thanks!

**Comment by zoultrex** [comment]
After a few more attempts it seems like this is an intermittent problem, I stopped the build after 55 minutes and started another one from command line, everything worked fine

Started another build from github action to deploy to a specific branch did not work, but starting a redeploy did

I'll fill in more details to try to nail down the possible cause later in next builds

**Comment by zoultrex** [comment]
After some more testing and I can say that builds do get stuck after `Cloning completed` when they are triggered from git push or merge. I just found a build stuck trying to build for over 15 hours and the only 2 entries again were just Cloning github, then cloning completed and nothing else.

Is there anything that I can do to debug this?

**Reply by samcx** [reply]
Is it possible to get more context on what you are trying to deploy on Vercel?

**Reply by zoultrex** [reply]
Sorry I replied below, was on my phone and just replied to the email which created a new reply below instead of creating the reply on this thread

**Comment by zoultrex** [comment]
Yes, it's the Redux Toolkit TypeScript Example, some more code on top of it
but the project structure it's exact as the template.

https://github.com/vercel/next.js/tree/canary/examples/with-redux

I'm still facing this issue, builds triggered by branch activity never pass
beyond post clone message, and stays forever there too, never fails or
timeout, last time I checked there was a build stuck for 4 days. I though
there would be a timeout...

**Reply by samcx** [reply]
https://github.com/vercel-support/with-redux-app → https://with-redux-app.vercel-support.app/ I was not able to replicate this issue @zoultrex
